packages/esm-patient-chart-app/src/visit/visit-action-items/edit-visit-details.component.tsx

```diff
@@ -22,10 +27,12 @@ const EditVisitDetailsActionItem: React.FC<EditVisitDetailsActionItemProps> = ({
     });
   };
 
+  const buttonLabel = compact ? getCoreTranslation('edit') : t('editVisitDetails', 'Edit visit details');
+
   return (
     <UserHasAccess privilege="Edit Visits">
       <Button onClick={editVisitDetails} kind="ghost" renderIcon={EditIcon} size={isTablet ? 'lg' : 'sm'}>
```

Comment:
I'd be inclined to use icon only buttons for inline actions (such as edit and delete) in line with [this guidance](https://carbondesignsystem.com/components/data-table/usage/#inline-actions) from the Carbon docs:

![CleanShot 2025-04-18 at 11  39 57@2x](https://github.com/user-attachments/assets/c007be8d-9063-480b-a37e-2cacd83876a9)